source/resampling_with_code2.Rmd

```diff
@@ -191,18 +194,19 @@ Notice that `0:5` means exactly "from 0 through 5" — the range includes all
 the integers starting at 0 and including 5.
 :::
 
-### Random numbers from 0 through 99
+::::
 
-We can now use {{< var np_or_r >}} and the random number function from the last chapter, to get 12 random numbers from 0 through 99.
+#### Random numbers from 0 through 99
+
+We can now use {{< var np_or_r >}} and the random number function from the last chapter to get 12 random numbers from 0 through 99.
 
 ```{python, opts.label="py_ed"}
 # Ask NumPy for a random number generator.
 rnd = np.random.default_rng()
 
 # Get 12 random numbers from 0 through 99
 # (from 0, up to, but not including 100).
-zero_through_99 = np.arange(0, 100)
-a = rnd.choice(zero_through_99, 12)
+a = rnd.integers(0, 100, size=12)
```

Comment:
Should we worry about keyword arguments here?   I mean, explaining them? Or should we defer to a later chapter?

Reply:
I think the code reads more clearly with them, but they should definitely be explained where we talk about functions.  For now, the reader will just have to infer that it tells the random integer tumbler to tumble out 12 numbers.